apps/openmw/mwrender/actoranimation.cpp

```diff
@@ -62,6 +62,7 @@ ActorAnimation::~ActorAnimation()
     }
 
     mScabbard.reset();
+    mHolsteredShield.reset();
```

Comment:
Is there a purpose to call reset here? C++ compilers generate code to autmatically destruct all members.

Reply:
Feel free to remove it if C++ will do it automatically.